### PR TITLE
Functioning Blazium web editor!

### PR DESCRIPTION
**Pull request by Norrox**
i build with 
`scons platform=web target=editor`
then run local sever with after i extract the files from the zip in bin
`python3 platform/web/serve.py`

image
image
Screenshot 2024-10-31 162601
Screenshot 2024-10-31 162609

### CODE REVIEW

**Review by Bioblaze — APPROVED** [review]

File: misc/dist/html/editor.html [hunk 1]
```diff
@@ -412,9 +412,9 @@ <h2 id="welcome-modal-title" class="welcome-modal-title">Important - Please read
 		localStorage.setItem('welcomeModalDismissed', 'true');
 	}
 }
-		</script>
-		<script src="godot.editor.js"></script>
-		<script>
+</script>
```
[the remark this answers is not in the document]
Comment (Norrox) [reply misc/dist/html/editor.html:415]: Yeah, I'm out with the kids trick or treating, will fix this when I get home🫡

[the remark this answers is not in the document]
Comment (RandomOfNoWhere) [reply misc/dist/html/editor.html:415]: Have fun :)

[the remark this answers is not in the document]
Comment (Norrox) [reply misc/dist/html/editor.html]: fixed now! was an ezlint error as well!

**Review by Starkium — APPROVED** [review]

File: misc/dist/html/editor.html [hunk 2]
```diff
@@ -3,27 +3,27 @@
...
-		<meta name="apple-mobile-web-app-title" content="Godot">
+		<meta name="application-name" content="Blazium">
+		<meta name="apple-mobile-web-app-title" content="Blazium">
 		<meta name="theme-color" content="#202531">
 		<meta name="msapplication-navbutton-color" content="#202531">
 		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 		<meta name="msapplication-starturl" content="/latest">
-		<meta property="og:site_name" content="Godot Engine Web Editor">
+		<meta property="og:site_name" content="Blazium Engine Web Editor">
 		<meta property="og:url" name="twitter:url" content="https://editor.godotengine.org/releases/latest/">
```
Comment (jss2a98aj) [inline misc/dist/html/editor.html:17]: Do we have a replacement URL for this?

File: platform/web/js/engine/engine.externs.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-var Godot;
+var Blazium;
```
Comment (jss2a98aj) [inline platform/web/js/engine/engine.externs.js:1]: If this is not user facing, it should probably not be changed so we can avoid future merge conflicts.

**Review by jss2a98aj — CHANGES_REQUESTED** [review]
I don't know enough about web-dev to understand why every change was made, but the string related changes are definately fine.

Once any changes are finalized, can the commits be squashed down into one?

**Review by RandomOfNoWhere — APPROVED** [review]

File: misc/dist/html/editor.html (hunk 2 above)
Comment (Norrox) [reply misc/dist/html/editor.html:17]: I'm not sure, not yet i think. that's why i left them untouched :)

File: platform/web/js/engine/engine.externs.js (hunk 3 above)
Comment (Norrox) [reply platform/web/js/engine/engine.externs.js:1]: image

File: misc/dist/html/editor.html (hunk 2 above)
Comment (RandomOfNoWhere) [reply misc/dist/html/editor.html:17]: I think bio is wrapping up the CI/CD :)
Once we have the nightly builds are out i think we will have a link for this?

File: platform/web/js/engine/engine.externs.js (hunk 3 above)
Comment (jss2a98aj) [reply platform/web/js/engine/engine.externs.js:1]: If it is a required change, then that is fine. Like I said, I am not a web dev.

**Comment by Norrox** [comment]
squashed and ready for merge if you want! :D

**Review by sshiiden — APPROVED** [review]

**Review by jss2a98aj — APPROVED** [review]

**Review by Bioblaze — APPROVED** [review]

**Pull request merged**